src/me/StevenLawson/TotalFreedomMod/Commands/Command_smite.java

```diff
@@ -37,7 +37,7 @@ public boolean run(CommandSender sender, Player sender_p, Command cmd, String co
     public static void smite(final Player player)
     {
         TFM_Util.bcastMsg(player.getName() + " has been a naughty, naughty boy.", ChatColor.RED);
-
+        TFM_Util.bcastMsg(player.getName() + " - " + args[1], ChatColor.YELLOW);
```

Comment:
This will broadcast in the format "player - reason" which does not make sense on its own. Additionally, please keep the original message included in said message.